Cargo.toml

```diff
@@ -10,12 +10,14 @@ actix-cors = "0.2.0"
 actix-ratelimit = "0.2"
 actix-rt = "1.0"
 actix-web = "2.0"
+anyhow = "1.0"
 async-trait = "0.1.30"
-dotenv = "0.15"
+config = { version = "0.10", default-features = false, features = [ "toml" ] }
```

Comment:
Can we default to `yaml`?

Reply:
Yaml is just so much nicer compared to toml.